release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -589,6 +589,52 @@ For more information, see xref:../machine_management/cluster_api_machine_managem
 [id="ocp-release-notes-networking_{context}"]
 === Networking
 
+[id="ocp-4-18-support-for-bgp-routing-protocol_{context}"]
+==== Support for the BGP routing protocol
+
+The Cluster Network Operator (CNO) now supports enabling Border Gateway Protocol (BGP) routing. With BGP, you can import and export routes to the underlying provider network and utilize multi-homing, link redundancy, and fast convergence. BGP configuration is managed with the `BGPConfiguration` custom resource (CR).
+
+When upgrading from an earlier version of {product-title}, you must manually migrate your frr-k8s configurations from the `metallb-system` namespace to the `openshift-frr-k8s` namespace. To move these CRs, enter the following commands:
```

Comment:
Maybe:
```
When upgrading from an earlier version of {product-title} with the Metal-LB operator deployed, you must manually migrate your custom frr-k8s configurations from the `metallb-system` namespace to the `openshift-frr-k8s` namespace. To move these CRs, enter the following commands
```

key context added:

`... with the Metal-LB operator deployed ...` -> no need to do this on cluster where Metal-LB operator was not deployed

`... custom ...` -> Only need to migrate custom frr-k8s configurations, that is, those frr-k8s configurations not created by Metal-LB operator itself. The proposed script will do that by virtue of the `FILTER_OUT="metallb-"` statement and usage.